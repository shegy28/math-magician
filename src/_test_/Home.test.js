import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Home from '../components/Home';
import '@testing-library/jest-dom';

test('renders Welcome to our page!', () => {
  render(<Home />);
  const linkElement = screen.getByText(/Welcome to our page!/i);
  expect(linkElement).toBeInTheDocument();
});

test('expect home component to match snapshot', () => {
  const view = renderer.create(<Home />).toJSON();
  expect(view).toMatchSnapshot();
});
