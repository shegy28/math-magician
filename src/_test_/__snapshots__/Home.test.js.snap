// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect home component to match snapshot 1`] = `
<div
  className="homepage"
>
  <h2>
     Welcome to our Home page! 
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam sed vestibulum erat. Nulla sagittis massa at egestas facilisis. Quisque tincidunt interdum vulputate. Sed orci orci, viverra vel tincidunt vel, consectetur eget tortor. Integer pellentesque eros vel ex maximus, quis aliquet nisl tristique. Nulla lobortis sem a placerat maximus. Duis suscipit purus at nunc bibendum, et finibus quam tempor. Donec sed magna suscipit, bibendum nisi faucibus, dictum erat. Aenean varius eros vitae arcu tempus lobortis. Ut et nisl eget purus ornare tincidunt. Aliquam erat volutpat. Suspendisse vitae finibus odio. Suspendisse aliquam risus massa, vel vulputate mauris tempus a.
  </p>
</div>
`;
